easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb

```diff
@@ -0,0 +1,2250 @@
+name = 'R'
+version = '3.3.3'
+x11ver = '20160819'
+versionsuffix = '-X11-%s' % x11ver
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['5ab768053a275084618fb669b4fbaadcc39158998a87e8465323829590bcfc6c']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=yes --enable-R-shlib"
```

Comment:
several of this stuff doesn't need to be here anymore, thanks to updates to the R easyblock, please see current (recent) R easyconfigs in `develop`